syntax = "proto3";
option java_package = "VireProtobuf.vire.cms";
option java_outer_classname = "InvalidUserErrorStub";
package vire.cms;

import "vire/utility/BaseError.proto";

message InvalidUserError {

  vire.utility.BaseError _base = 1;
  oneof case_oneof {
    string invalid_user_name = 100; // Invalid user name
    sint32 invalid_user_id   = 101; // Invalid UID
  }

}